consensus/module.go

```diff
@@ -1,37 +1,44 @@
 package consensus
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 
 	"github.com/pokt-network/pocket/consensus/leader_election"
-	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
 	typesCons "github.com/pokt-network/pocket/consensus/types"
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
-	"github.com/pokt-network/pocket/shared/modules"
-	"github.com/pokt-network/pocket/shared/types"
-	"github.com/pokt-network/pocket/shared/types/genesis"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
+
+	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
+	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
-	DefaultLogPrefix string = "NODE" // Just a default that'll be replaced during consensus operations.
+	DefaultLogPrefix    string = "NODE" // Just a default that'll be replaced during consensus operations.
+	ConsensusModuleName        = "consensus"
+	PacemakerModuleName        = "pacemaker"
```

Comment:
I think we should put this in `pacemaker.go`. Doesn't make sense for it to be here